modules/mobile/app/services/mobile/v0/claims/proxy.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'lighthouse/facilities/client'
 
+# [wipn8923] is this a valid use case?  if so, do we need to update or replace it?
```

Comment:
It looks like this was replaced here https://github.com/department-of-veterans-affairs/vets-api/blob/dbex-64564-lighthouse-supporting-evidence-attachment/modules/mobile/app/services/mobile/v0/lighthouse_claims/proxy.rb, but I don't think it applies to our form, I think this is a separate mobile claims tool